orttraining/orttraining/training_ops/cuda/optimizer/lamb.h

```diff
@@ -126,7 +126,17 @@ struct LambMultiTensorComputeDirectionFunctor {
 //  d_norm: chunk_group.tensor_ptrs[3][i]
 template <typename TIn1, typename TIn2, typename TOut1, typename TOut2, typename TBuf>
 struct LambMultiTensorReductionFunctor {
-  void operator()(ChunkGroup<4> chunk_group);
+  void operator()(
+    ChunkGroup<4> chunk_group, 
+    const CudaKernel& kernel, 
+    void *reduction_buffer, 
+    size_t reduction_buffer_size);
+};
+
+struct LambMultiTensorSyncRangeAndLock {
```

Comment:
>LambMultiTensorSyncRangeAndLock [](start = 7, length = 31)

Need a document for this struct like others in this file.

Reply:
OK, added. Tried to be clear. Should suffice I think.